README.md

```diff
@@ -34,12 +34,16 @@ You then need to load the plugin. In `boostrap.php`, something like:
 \Cake\Core\Plugin::load('BootstrapUI');
 ```
 
-For a complete setup, add the following to your `App\Controller\AppController`:
+For a complete setup, add the following to your `App\View\AppView`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
-public $helpers = ['BootstrapUI.Form'];
 public $layout = 'BootstrapUI.default';
```

Comment:
Can one change the layout value in a controller if only defined in `AppView`? I usually define `$layout` in `AppController` and the load the helpers with `AppView`. Maybe am doing it wrong?